src/OI.cpp

```diff
@@ -61,23 +61,23 @@ OI::OI() {
     dButtonR3.reset(new JoystickButton(driver.get(), 10));
     dButtonR3->WhileHeld(new AutonomousCommand());
     dButtonL3.reset(new JoystickButton(driver.get(), 9));
-    dButtonL3->WhenReleased(new ShifterToggle());
+    dButtonL3->WhenPressed(new ShifterToggle());
```

Comment:
WhenReleased is just odd. Most people expect the button to do something when it goes down, not when it comes up. (The exception is a mouse button, but that's because a mouse can move.)